src/generators/layout.fsx

```diff
@@ -92,9 +92,9 @@ let layout (ctx : SiteContents) active bodyCnt =
             link [Rel "stylesheet"; Type "text/css"; Href (urlPrefix + "/style/style.css")]
             link [Rel "stylesheet"; Type "text/css"; Href "https://cdn.jsdelivr.net/npm/@creativebulma/bulma-collapsible@1.0.4/dist/css/bulma-collapsible.min.css"]
             
-            script [ Defer true; Src "https://kit.fontawesome.com/0d3e0ea7a6.js"; CrossOrigin "anonymous"][]
+            script [ Defer true; Src "https://kit.fontawesome.com/0d3e0ea7a6.js"; CrossOrigin "anonymous"] []
```

Comment:
This was required since I was getting some warnings/errors when compiling this project locally.